test/v1/connectors/RetrieveEmploymentExpensesConnectorSpec.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.connectors
+
+import mocks.MockAppConfig
+import uk.gov.hmrc.domain.Nino
+import v1.mocks.MockHttpClient
+import v1.models.domain.MtdSource
+import v1.models.outcomes.ResponseWrapper
+import v1.models.request.retrieveEmploymentExpenses.RetrieveEmploymentsExpensesRequest
+import v1.models.response.retrieveEmploymentExpenses.{Expenses, RetrieveEmploymentExpensesResponse}
+
+import scala.concurrent.Future
+
+class RetrieveEmploymentExpensesConnectorSpec extends ConnectorSpec {
+
+  private val nino = Nino("AA123456A")
+  private val taxYear = "2019-20"
+  val source = MtdSource.`user`
+
+
+
+  class Test extends MockHttpClient with MockAppConfig {
+    val connector: RetrieveEmploymentExpensesConnector = new RetrieveEmploymentExpensesConnector(http = mockHttpClient, appConfig = mockAppConfig)
+    val desRequestHeaders: Seq[(String, String)] = Seq("Environment" -> "des-environment", "Authorization" -> s"Bearer des-token")
+    MockedAppConfig.desBaseUrl returns baseUrl
+    MockedAppConfig.desToken returns "des-token"
+    MockedAppConfig.desEnvironment returns "des-environment"
+  }
+
+  "retrieve employment expenses" should {
+    val request = RetrieveEmploymentsExpensesRequest(nino, taxYear, source)
+    "return a result" when {
+      "the downstream call is successful" in new Test {
+        val outcome = Right(ResponseWrapper(correlationId, RetrieveEmploymentExpensesResponse(
+          Some("2019-04-06"),
+          Some(2000.99),
+          Some(MtdSource.`user`),
+          Some("2019-04-06"),
+          Some(Expenses(
+            Some(2000.99),
+            Some(2000.99),
+            Some(2000.99),
+            Some(2000.99),
+            Some(2000.99),
+            Some(2000.99),
+            Some(2000.99),
+            Some(2000.99)
+          ))
+        )))
+        MockedHttpClient.get(
+          url = s"$baseUrl/income-tax/expenses/employments/${request.nino}/${request.taxYear}?view=${request.source.toDes}",
```

Comment:
can we add the actual values here not just the same thing as the code. otherwise how do we know whats in the code is doing what it's ment to